### ISSUE TITLE
npm test failure on eslint

### ISSUE THREAD

**Issue by YasharF** [ryanwiemer/gatsby-starter-gcn#66]
The eslint rule is "Disallow shorthand properties that override related longhand properties.".

The test failure is by:
https://github.com/ryanwiemer/gatsby-starter-gcn/commit/1514df5ffdfd86506a78a93f977e59b6c71a7dfa#diff-c8a07c8225c0af21bdd6c40b67170518R26

More about the eslint rule: https://stylelint.io/user-guide/rules/declaration-block-no-shorthand-property-overrides/

```
$ npm run test

> gatsby-starter-gcn@2.0.5 test /mnt/c/temp/gatsby-starter-gcn
> npm run lint:css && npm run lint:js && npm run check-pretty


> gatsby-starter-gcn@2.0.5 lint:css /mnt/c/temp/gatsby-starter-gcn
> stylelint 'src/**/*.js'


src/styles/global.js
 25:5  ✖  Unexpected shorthand "font"   declaration-block-no-shorthand-property-overrides
          after "font-size"

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! gatsby-starter-gcn@2.0.5 lint:css: `stylelint 'src/**/*.js'`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the gatsby-starter-gcn@2.0.5 lint:css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2019-06-15T17_41_23_017Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! gatsby-starter-gcn@2.0.5 test: `npm run lint:css && npm run lint:js && npm run check-pretty`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the gatsby-starter-gcn@2.0.5 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2019-06-15T17_41_23_058Z-debug.log
```

**Comment by ryanwiemer** [comment]
Thank you for the detailed info. I will take a look at this.

**Comment by ryanwiemer** [comment]
@YasharF This should be resolved as of [`v2.0.6`](https://github.com/ryanwiemer/gatsby-starter-gcn/releases/tag/v2.0.6)
